docs/common-coreboot-docs/checkbox.md

```diff
@@ -0,0 +1,227 @@
+# Checkbox usage (Ubuntu 22.04)
+
+## Introduction
+
+Checkbox is a system testing platform for Ubuntu. It aims to provide a common
+framework to run all types of tests, from hardware tests, to command line tests,
+unit tests or desktop tests. This document describes the usage of Checkbox on
+Ubuntu 22.04, but it can work on older versions of Ubuntu.
+
+## Prerequisites
+
+To achieve the best results from the tests and do many of them, we should fill
+as many internal and external slots as possible. Before running the checkbox,
+try to fill the slots if the DUT has one:
+
+External:
+* Memory Card Reader
+* USB Ports
+* Monitor Ports
+* Ethernet Ports
+* Headphone and Microphone Jacks
+* Thunderbolt Ports
+* Power Socket
+* PCMCIA or ExpressCard slot
+
+Internal:
+* DIMM/RAM slots
+* SATA ports
+* PCI/PCIe slots
+* M.2 slots
+* TPM header
+* Other slots
+
+## Set up testing environment
+
+In order for the tests to not require any interference after their startup,
+perform the following steps:
+
+To stop the screen from being locked on suspend, open the terminal and run the
+following command:
+
+```bash
+ gsettings set org.gnome.desktop.lockdown disable-lock-screen 'true'
+```
+
+To execute all sudo commands without password, open the terminal and run the
+following command:
+
+```bash
+sudo visudo
+```
+
+Append the following entry at the end of file to run ALL commands without a
+password for a user:
+
+```bash
+user ALL=(ALL) NOPASSWD:ALL
+```
+
+To set up automatic login so that a password is not required during testing
+reboot, poweroff etc.:
+
+1. Press the `SUPER_KEY`, type in `Users` and click Enter. Window with options to
+change should appear.
+1. Next click the user name you want to enable and disable automatic login.
+1. Then on the top right side of the window click `Unlock...` button.
+1. You will be required to enter your user password to continue.
+1. You should now be able to toggle Automatic Login button to enable.
+1. Restart your system for changes to take effect.
```

Comment:
Probably one or two screenshots would be helpful when trying to explain it the GUI-way.

But, OTOH, it is quite possible that using CLI would also be possible here to simplify the setup by pasting some commands.

Reply:
added one image